### PR TITLE
Collect sizes in `MsgListDatabases` for Tigris

### LINKED ISSUE #776: Collect sizes in `MsgListDatabases` for Tigris
https://www.mongodb.com/docs/manual/reference/command/listDatabases/

https://pkg.go.dev/github.com/tigrisdata/tigris-client-go@v1.0.0-alpha.22/api/server/v1#DescribeDatabaseResponse

Update an existing regular integration test, do not add compat test.

### PR DESCRIPTION
**Pull request by w84thesun**
# Description

This PR closes #776.

## Readiness checklist

* [x] I added tests for new functionality or bugfixes.
* [x] I ran `task all`, and it passed.
* [x] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [x] I checked comments rendering with `task godocs`.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by rumyantseva — CHANGES_REQUESTED** [review]
Sorry, this was not for this PR :0

**Review by noisersup — DISMISSED** [review]
LGTM

File: integration/commands_administration_test.go [hunk 1]
```diff
@@ -853,3 +853,28 @@ func TestCommandsAdministrationWhatsMyURI(t *testing.T) {
 	require.Equal(t, 2, len(ports))
 	assert.NotEqual(t, ports[0], ports[1])
 }
+
+func TestCommandsAdministrationListDatabases(t *testing.T) {
```
Comment (AlekSi) [inline integration/commands_administration_test.go:857]: Please move it after `TestCommandsAdministrationCreateDropListDatabases`

Comment (w84thesun) [reply integration/commands_administration_test.go]: Done, plus changed from `SkipForTigris` to `SkipForTigrisWithReason`.

**Review by noisersup — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM

File: internal/handlers/tigris/msg_listdatabases.go [hunk 2]
```diff
@@ -58,7 +58,7 @@ func (h *Handler) MsgListDatabases(ctx context.Context, msg *wire.OpMsg) (*wire.
 		// iterate over result to collect sizes
 		var sizeOnDisk int64
 		for _, c := range res.Collections {
-			_ = c // TODO https://github.com/FerretDB/FerretDB/issues/776
+			sizeOnDisk += c.Size
```
Comment (AlekSi) [inline internal/handlers/tigris/msg_listdatabases.go:61]: DescribeDatabase now returns size directly, no need to iterate over collections

**Review by AlekSi — CHANGES_REQUESTED** [review]

**Comment by AlekSi** [comment]
> Add collections sizes to `MsgListDatabases` response for Tigris

That method should return database sizes, not collection sizes

**Review by AlekSi — APPROVED** [review]

File: internal/handlers/tigris/tigrisdb/errors.go [hunk 3]
```diff
@@ -22,25 +22,21 @@ import (
 
 // IsNotFound returns true if the error is "not found" error.
 // This function is implemented to keep nolint in a single place.
-func IsNotFound(err *driver.Error) bool {
-	//nolint:nosnakecase // Tigris named their const that way
-	return pointer.Get(err).Code == api.Code_NOT_FOUND
+func IsNotFound(err error) bool {
+	e, _ := err.(*driver.Error)
```
Comment (rumyantseva) [inline internal/handlers/tigris/tigrisdb/errors.go:26]: That's handy! 👍

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**